test/test_other.py

```diff
@@ -319,6 +319,7 @@ def test_emcc_output_worker_mjs(self, args):
     src = read_file('subdir/hello_world.mjs')
     self.assertContained("new URL('hello_world.wasm', import.meta.url)", src)
     self.assertContained("new Worker(new URL('hello_world.worker.js', import.meta.url), {type: 'module'})", src)
+    self.assertContained("new Worker(pthreadMainJs, {type: 'module'})", src)
```

Comment:
In general I think it would be preferable to check the behaviour, rather than the precise output... but maybe its not easy in this case.

Is it possible to reproduce the failure in question here?

Reply:
The failure was only for Deno runtime which I don't think test suite currently runs with. Maybe it should, but it would be bigger and out of scope of this PR.

Reply:
sgtm.   

lgtm